pkgs/by-name/za/zapzap/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zapzap";
+  version = "4.5.5.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "zapzap-linux";
+    repo = "zapzap";
+    rev = version;
+    hash = "sha256-8IeFGTI+5kbeFGqH5DpHCY8pqzGhE48hPCEIKIe7jAM=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ];
+
```

```suggestion

  buildInputs = [
    qt6.qtwayland
    qt6.qtsvg
  ];

  preBuild = ''
    export HOME=$(mktemp -d)
  '';

```

qtsvg is required to properly display the icons